packages/devtools_app/pubspec.yaml

```diff
@@ -163,6 +163,9 @@ flutter:
     - family: Octicons
       fonts:
         - asset: fonts/Octicons.ttf
+    - family: Noto
+      fonts:
+        - asset: fonts/Noto/NotoMusic-Regular.ttf
```

Comment:
before we add this we should make sure this already exists in g3 and will not block our roll.